src/Generator/Template/PropertyInitializationTemplate.php

```diff
@@ -9,12 +9,12 @@
  */
 class PropertyInitializationTemplate implements TemplateInterface
 {
-    const TEMPLATE = '
+    private const TEMPLATE = '
             unset(%s%s);
     ';
 
     /**
-     * @param \ReflectionProperty $property
+     * @param \Reflector|\ReflectionProperty $property
```

Comment:
Same as above.

Reply:
Because we call a specific method of the concrete class implementation.